src/stdio.rs

```diff
@@ -4,13 +4,15 @@ use super::Error;
 use super::native_mux_impl;
 
 use io_lifetimes::OwnedFd;
+use std::convert::TryFrom;
+use std::convert::TryInto;
```

Comment:
These two should no longer be necessary in the 2021 edition, which I think it's okay for us to move to as part of 0.9.0

Reply:
Done.